recipes/hello_tests/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# A temporary hack.
+source activate _build
```

Comment:
A cleaner strategy as it may not always be `_build` would be to do the following.

```
source activate "${CONDA_DEFAULT_ENV}"
```